### PR TITLE
WRN-12963: Add back button to Video Player

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Video player needs backbutton and a prop for aria-label override.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Cherrypick https://github.com/enactjs/sandstone/pull/803 
- Add backButtonAriaLabel.
- To support RTL, add iconFlip="auto"

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-12963, WRN-127111

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: MediaPlayer/MediaControls.js [hunk 1]
```diff
@@ -26,7 +26,8 @@ import css from './MediaControls.module.less';
 const OuterContainer = SpotlightContainerDecorator({
 	defaultElement: [
 		`.${spotlightDefaultClass}`
-	]
+	],
+	leaveFor: {left: '', right: ''}
```
Comment (jeonghee27) [inline MediaPlayer/MediaControls.js:30]: https://github.com/enactjs/sandstone/pull/803/files#r530757813

File: VideoPlayer/VideoPlayer.js [hunk 2]
```diff
@@ -84,7 +85,7 @@ const RootContainer = SpotlightContainerDecorator(
 
 const ControlsContainer = SpotlightContainerDecorator(
 	{
-		enterTo: '',
+		enterTo: 'default-element',
```
Comment (jeonghee27) [inline VideoPlayer/VideoPlayer.js:88]: Move focus to "the slider" from "the back button".

File: VideoPlayer/VideoPlayer.js [hunk 3]
```diff
@@ -1836,8 +1849,6 @@ const VideoPlayerBase = class extends Component {
 			}
 		} else if (is('up', keyCode)) {
 			Spotlight.setPointerMode(false);
-			preventDefault(ev);
-			stopImmediate(ev);
```
Comment (jeonghee27) [inline VideoPlayer/VideoPlayer.js:1840]: To get the back button using "up" button

File: styles/variables.less [hunk 4]
```diff
@@ -943,6 +943,8 @@
 
 // Video
 // ---------------------------------------
+@sand-video-back-button-left                      : 150px;
+@sand-video-back-button-top                       : 114px;
```
Comment (MikyungKim) [inline styles/variables.less:947]:
```suggestion
@sand-video-back-button-left :                                        150px;
@sand-video-back-button-top :                                        114px;
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,12 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ## [unreleased]
 
+
```
Comment (MikyungKim) [inline CHANGELOG.md:7]: Please remoe this line.

File: VideoPlayer/VideoPlayer.js (hunk 2 above)
Comment (MikyungKim) [reply VideoPlayer/VideoPlayer.js:88]: 🆗

File: MediaPlayer/MediaControls.js (hunk 1 above)
Comment (MikyungKim) [reply MediaPlayer/MediaControls.js:30]: 🆗

File: VideoPlayer/VideoPlayer.js (hunk 2 above)
Comment (MikyungKim) [reply VideoPlayer/VideoPlayer.js:88]: 🆗

Comment (MikyungKim) [reply VideoPlayer/VideoPlayer.js:88]: For the record, you can check the behavior by entering focus via 5way from the back button to controls.

File: styles/variables.less (hunk 4 above)
Comment (jeonghee27) [reply styles/variables.less]: I fixed it.

File: CHANGELOG.md (hunk 5 above)
Comment (jeonghee27) [reply CHANGELOG.md]: I fixed it.

File: VideoPlayer/VideoPlayer.js [hunk 6]
```diff
@@ -2031,6 +2044,18 @@ const VideoPlayerBase = class extends Component {
...
 							{secondsToTime(this.state.sliderTooltipTime, durFmt)}
 						</FeedbackContent>
+						{
+							this.state.mediaControlsVisible ?
+								<Button
+									aria-label={backButtonAriaLabel == null ? $L('go to previous') : backButtonAriaLabel}
+									className={css.back}
+									icon="arrowhookleft"
+									iconFlip="auto"
+									onClick={onBack}
```
Comment (MikyungKim) [inline VideoPlayer/VideoPlayer.js:2054]: This lets the event type not be `onBack`.
Let's use `this.handleBack` which look like
```
handleBack = this.handle(forwardCustom('onBack'));
```

**Review by MikyungKim — APPROVED** [review]
LGTM :shipit:

**Pull request merged**